src/main/java/de/kekru/struktogrammeditor/view/CustomJButton.java

```diff
@@ -0,0 +1,47 @@
+package de.kekru.struktogrammeditor.view;
+
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+
+import javax.swing.JButton;
+import javax.swing.JOptionPane;
+import de.kekru.struktogrammeditor.control.Struktogramm;
+
+public class CustomJButton extends JButton {
```

Comment:
What does this Button do? Maybe you find a name that tells us, what it does, e.g. `TabbedPaneClosingButton`

Reply:
You can delete this one now, right? Because you added `TabbedPaneClosingButton`